### PR TITLE
VictorMurashev-homework-20240804

### PR DESCRIPTION
**Pull request by vmurashev365**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -121,7 +121,7 @@
         <dependency>
             <groupId>com.google.code.gson</groupId>
             <artifactId>gson</artifactId>
-            <version>2.8.0</version>
+            <version>2.8.9</version>
```
Comment (dgrimaliuc-crunchyroll) [inline pom.xml:124]: don't touch versions, return it back

Comment (vmurashev365) [reply pom.xml]: > don't touch versions, return it back

@dgrimaliuc done

File: src/test/java/denis_grimaliuc/step_definitions/UIStepDefinition.java [hunk 2]
```diff
@@ -60,17 +60,17 @@ public void verifyURLOfPage() {
     }
 
     @Given("Shopify Page is Open")
-    public void shopifyPageIsOpen() {
+    public void shopify_page_is_open() {
```
Comment (dgrimaliuc-crunchyroll) [inline src/test/java/denis_grimaliuc/step_definitions/UIStepDefinition.java:63]: my file should not be changed

**Review by dgrimaliuc-crunchyroll — CHANGES_REQUESTED** [review]

**Comment by dgrimaliuc** [comment]
please resolve conflicts

**Review by dgrimaliuc-crunchyroll — DISMISSED** [review]

**Pull request closed without merging**